packages/@react-spectrum/color/test/ColorWheel.test.tsx

```diff
@@ -0,0 +1,299 @@
+      let defaultColor = new Color('hsl(0, 100%, 50%)');
+      let {getByRole} = render(<ColorWheel defaultValue={defaultColor} onChange={onChangeSpy} isDisabled />);
+      let slider = getByRole('slider');
+      act(() => {slider.focus();});
+
+      fireEvent.keyDown(slider, {key: 'Right'});
+      expect(onChangeSpy).toHaveBeenCalledTimes(0);
+      fireEvent.keyDown(slider, {key: 'Left'});
+      expect(onChangeSpy).toHaveBeenCalledTimes(0);
+    });
+
+    it('wraps around', () => {
+      let defaultColor = new Color('hsl(0, 100%, 50%)');
+      let {getByRole} = render(<ColorWheel defaultValue={defaultColor} onChange={onChangeSpy} />);
+      let slider = getByRole('slider');
+      act(() => {slider.focus();});
+
+      fireEvent.keyDown(slider, {key: 'Left'});
+      expect(onChangeSpy).toHaveBeenCalledTimes(1);
+      expect(onChangeSpy.mock.calls[0][0].toString('hsla')).toBe(defaultColor.withChannelValue('hue', 359).toString('hsla'));
+    });
+
+    it('respects step', () => {
+      let defaultColor = new Color('hsl(0, 100%, 50%)');
+      let {getByRole} = render(<ColorWheel defaultValue={defaultColor} onChange={onChangeSpy} step={45} />);
+      let slider = getByRole('slider');
+      act(() => {slider.focus();});
+
+      fireEvent.keyDown(slider, {key: 'Right'});
+      expect(onChangeSpy).toHaveBeenCalledTimes(1);
+      expect(onChangeSpy.mock.calls[0][0].toString('hsla')).toBe(defaultColor.withChannelValue('hue', 45).toString('hsla'));
+      fireEvent.keyDown(slider, {key: 'Left'});
+      expect(onChangeSpy).toHaveBeenCalledTimes(2);
+      expect(onChangeSpy.mock.calls[1][0].toString('hsla')).toBe(defaultColor.withChannelValue('hue', 0).toString('hsla'));
+    });
+  });
+
+  describe.each`
+    type                | actions                                                                 | prepare
+    ${'Mouse Events'}   | ${[fireEvent.mouseDown, fireEvent.mouseMove, fireEvent.mouseUp]}        | ${installMouseEvent}
+    ${'Pointer Events'} | ${[fireEvent.pointerDown, fireEvent.pointerMove, fireEvent.pointerUp]}  | ${installPointerEvent}
+    ${'Touch Events'}   | ${[
+      (el, {pageX, pageY}) => fireEvent.touchStart(el, {targetTouches: [{pageX, pageY}]}),
+      (el, {pageX, pageY}) => fireEvent.touchMove(el, {targetTouches: [{pageX, pageY}]}),
+      (el, {pageX, pageY}) => fireEvent.touchEnd(el, {targetTouches: [{pageX, pageY}]})
+    ]}      | ${() => {}}
+  `('$type', ({actions: [start, move, end], prepare}) => {
+    prepare();
+
+    it('dragging the thumb works', () => {
+      let defaultColor = new Color('hsl(0, 100%, 50%)');
+      let {container: _container, getByRole} = render(<ColorWheel defaultValue={defaultColor} onChange={onChangeSpy} />);
+      let container = _container.firstChild as HTMLElement;
+      let thumb = container.querySelector('.spectrum-ColorWheel-handle') as HTMLElement;
+      let loupe = thumb.querySelector('.spectrum-ColorLoupe') as HTMLElement;
```

Comment:
We usually try to keep the queries semantic rather than relying on class names. Maybe add testids to these elements instead?